app/src/main/kotlin/be/zvz/kookie/item/ItemFactory.kt

```diff
@@ -33,9 +39,303 @@ object ItemFactory {
         register(Apple(ItemIdentifier(ItemIds.APPLE.id, 0), "Apple"))
         register(Arrow(ItemIdentifier(ItemIds.ARROW.id, 0), "Arrow"))
 
+        register(BakedPotato(ItemIdentifier(ItemIds.BAKED_POTATO.id, 0), "Baked Potato"))
+        register(Bamboo(ItemIdentifier(ItemIds.BAMBOO.id, 0), "Bamboo"), true)
+        register(Beetroot(ItemIdentifier(ItemIds.BEETROOT.id, 0), "Beetroot"))
+        register(BeetrootSeeds(ItemIdentifier(ItemIds.BEETROOT_SEEDS.id, 0), "Beetroot Seeds"))
+        register(BeetrootSoup(ItemIdentifier(ItemIds.BEETROOT_SOUP.id, 0), "Beetroot Soup"))
+        register(BlazeRod(ItemIdentifier(ItemIds.BLAZE_ROD.id, 0), "Blaze Rod"))
+        register(Book(ItemIdentifier(ItemIds.BOOK.id, 0), "Book"))
+        register(Bow(ItemIdentifier(ItemIds.BOW.id, 0), "Bow"))
+        register(Bowl(ItemIdentifier(ItemIds.BOWL.id, 0), "Bowl"))
+        register(Bread(ItemIdentifier(ItemIds.BREAD.id, 0), "Bread"))
+        register(Bucket(ItemIdentifier(ItemIds.BUCKET.id, 0), "Bucket"))
+        register(Carrot(ItemIdentifier(ItemIds.CARROT.id, 0), "Carrot"))
+        register(ChorusFruit(ItemIdentifier(ItemIds.CHORUS_FRUIT.id, 0), "Chorus Fruit"))
+        register(Clock(ItemIdentifier(ItemIds.CLOCK.id, 0), "Clock"))
+        register(Clownfish(ItemIdentifier(ItemIds.CLOWNFISH.id, 0), "Clownfish"))
+        register(Coal(ItemIdentifier(ItemIds.COAL.id, 0), "Coal"))
+
+        // TODO: how?
```

Comment:
What?

Reply:
It should be fixed after add CoralBlock in `feature/block` branch.